lib/client/mfa/cli.go

```diff
@@ -35,117 +38,155 @@ import (
 	"github.com/gravitational/teleport/lib/auth/webauthnwin"
 )
 
+// CLIMFAType is the CLI display name for an MFA type.
+type CLIMFAType string
+
+const (
+	CLIMFATypeOTP      = "OTP"
+	CLIMFATypeWebauthn = "WEBAUTHN"
+	CLIMFATypeSSO      = "SSO"
```

Comment:
```suggestion
	CLIMFATypeOTP      CLIMFATypeOTP = "OTP"
	CLIMFATypeWebauthn CLIMFATypeOTP = "WEBAUTHN"
	CLIMFATypeSSO      CLIMFATypeOTP = "SSO"
```

otherwise it doesn't really get the type.

Reply:
I think I'll actually just remove the type as we want to do things like `strings.Join()` on them.

Reply:
Sure, that's fine too. Any chance we could make them private?